engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/encode/ValidationMode.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.engine.avro.encode
+
+case class ValidationMode private(name: String, label: String, acceptUnfilledOptional: Boolean, acceptRedundant: Boolean)
+
+object ValidationMode {
+
+  val strict: ValidationMode = ValidationMode("strict", "Strict mode", acceptUnfilledOptional = false, acceptRedundant = false)
```

Comment:
these all static field should be also final ?

Reply:
@lciolecki In scala, final modifier for object's vals do nothing. `final` mean "won't be overridden" `val` mean "is contant"